.editorconfig

```diff
@@ -3,11 +3,8 @@ root = true
 
 [*]
 indent_style = space
-indent_size = 4
+indent_size = 2
```

Comment:
because 4 space indents are H U G E !

https://github.com/guardian/grid/pull/2465/files?w=1 may be friendlier to review